### PR TITLE
loc updates

### PR DESCRIPTION
**Pull request by cristianosuzuki77**
Several updates to previous translations.

### CODE REVIEW

File: src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf [hunk 1]
```diff
@@ -234,7 +234,7 @@
       </trans-unit>
       <trans-unit id="ByteRangeDownloaderDisposed">
         <source>Cannot access a disposed HTTP byte range downloader.</source>
-        <target state="translated">無法存取已處置的 HTTP 位元組範圍下載器。</target>
+        <target state="translated">無法存取已處置的 HTTP 位元組範圍下載程式。</target>
```
Comment (lindexi) [inline src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf:237]: You translate the `downloader` as `下载器` in zh-Hans but you translate it as `下載程式` to replace `下載器`(`下载器`) in zh-Hant. And I do not think it is a good idea. But in Chinese, both translation methods are right.

File: src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf [hunk 2]
```diff
@@ -1069,7 +1069,7 @@
       </trans-unit>
       <trans-unit id="Freezable_CloneInvalidType">
         <source>Clone of an instance of type '{0}' is null or not an instance of '{0}'.</source>
-        <target state="translated">型別 '{0}' 的執行個體複本是空值，或不是 '{0}' 的執行個體。</target>
+        <target state="translated">型別 '{0}' 的執行個體複製品是空值，或不是 '{0}' 的執行個體。</target>
```
Comment (lindexi) [inline src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf:1072]: I do not think we should translate `Clone` as `複製品`.

In this context, it should be translated as a `複本`

File: src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Resources/xlf/Strings.zh-Hant.xlf [hunk 3]
```diff
@@ -154,7 +154,7 @@
       </trans-unit>
       <trans-unit id="InvalidAssemblyVersion">
         <source>MC1005: Invalid AssemblyVersion detected: {0}.</source>
-        <target state="translated">MC1005：偵測到無效的 AssemblyVersion: {0}。</target>
+        <target state="translated">MC1005:偵測到無效的 AssemblyVersion: {0}.</target>
```
Comment (lindexi) [inline src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Resources/xlf/Strings.zh-Hant.xlf:157]: This punctuation is misalmed and should be Chinese the same time

```suggestion
        <target state="translated">MC1005：偵測到無效的 AssemblyVersion: {0}。</target>
```

**Review by lindexi — CHANGES_REQUESTED** [review]
Thank you. There are some inappropriate Chinese translations.

File: src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf (hunk 2 above)
Comment (cristianosuzuki77) [reply src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf:1072]: @lindexi  複製品 is correct according to the term, per Microsoft.
複本 is for another term.

File: src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Resources/xlf/Strings.zh-Hant.xlf (hunk 3 above)
Comment (cristianosuzuki77) [reply src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Resources/xlf/Strings.zh-Hant.xlf]: @lindexi Thank you. Updated the file and am requesting the translators to review the translation memories.

File: src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf (hunk 1 above)
Comment (cristianosuzuki77) [reply src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf:237]: I'm asking the translators to review your comment.

File: src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf (hunk 2 above)
Comment (lindexi) [reply src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf:1072]: @cristianosuzuki77 Thank you

**Comment by ryalanms** [comment]
Thanks, @lindexi.  @cristianosuzuki77:  Feel free to merge any loc changes in WPF that you approve of.  Thanks.

**Comment by ryalanms** [comment]
@cristianosuzuki77: Feel free to merge when the translators approve.  Thanks.

**Comment by ryalanms** [comment]
@cristianosuzuki77:  Can we take this change? Thanks.

**Comment by crsuzukimsft** [comment]
Apologies. I'm still waiting on the feedback from translators. Should be within the next day.

**Comment by crsuzukimsft** [comment]
Just got a confirmation that the translator will provide a better translation. I will update the file and re-submit.

File: src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf (hunk 1 above)
Comment (crsuzukimsft) [reply src/Microsoft.DotNet.Wpf/src/PresentationCore/Resources/xlf/Strings.zh-Hant.xlf:237]: Confirmed with LM that current translation "無法存取已處置的 HTTP 位元組範圍下載程式" is correct.

**Comment by ryalanms** [comment]
@crsuzukimsft:  It looks like all the comments are resolved.  Is it okay to merge? 

/cc @fabiant3

**Comment by cristianosuzuki77** [comment]
Yes, good to merge.

**Pull request merged**